app/main/controllers/template/rtmedia-ajax-actions.php

```diff
@@ -16,18 +17,50 @@ function rtmedia_delete_uploaded_media() {
 
 	if ( ! empty( $action ) && 'delete_uploaded_media' === $action && ! empty( $media_id ) ) {
 		if ( wp_verify_nonce( $nonce, 'rtmedia_' . get_current_user_id() ) ) {
+			$remaining_photos    = 0;
+			$remaining_music     = 0;
+			$remaining_videos    = 0;
+			$remaining_all_media = 0;
+			$rtmedia_media       = new RTMediaMedia();
+			$rtmedia_media->delete( $media_id );
+
+			// Fetch the remaining media count.
+			if ( class_exists( 'RTMediaNav' ) ) {
+				global $bp;
+				$rtmedia_nav_obj = new RTMediaNav();
+
+				if ( function_exists( 'bp_is_group' ) && bp_is_group() && isset( $bp->groups->current_group->id ) && ! empty( $bp->groups->current_group->id ) ) {
```

Comment:
If you're checking `empty` then no need to check `isset`.

Reply:
Move `! empty( $bp->groups->current_group->id )` inside.
The whole code would look like this
```
if ( function_exists( 'bp_is_group' ) && bp_is_group() ) {

	if ( ! empty( $bp->groups->current_group->id ) ) {
		$counts = $rtmedia_nav_obj->actual_counts( $bp->groups->current_group->id, 'group' );
	}

} elseif ( function_exists( 'bp_displayed_user_id' ) ) {
	$counts = $rtmedia_nav_obj->actual_counts( bp_displayed_user_id(), 'profile' );
}
```
If bp_is_group() exists and they change `$bp->groups->current_group->id` only, this will check for `elseif` and if it gets true, this will show wrong count (user's count instead of group count). It's fine if it shows only 0, it shouldn't show wrong count.

Reply:
`if` block changed.